scripts/concurrent_task_utils.py

```diff
@@ -103,17 +103,17 @@ def _check_all_tasks(tasks):
             log(task_details)
 
 
-def execute_tasks(tasks, semaphore=SEMAPHORE):
+def execute_tasks(tasks, max_concurrent_runs):
     """Starts all tasks and checks the results.
     Runs no more than the allowable limit defined in the semaphore.
 
     Args:
         tasks: list(TestingTaskSpec). The tasks to run.
-        semaphore: threading.Semaphore. The object that controls how many tasks
-            can run at any time.
+        max_concurrent_runs: int. Number of tasks that run at a time.
```

Comment:
Number --> Maximum number
run --> are allowed to run

Reply:
This was removed in https://github.com/oppia/oppia/pull/8656/commits/c0ded58aeb9b9c01ff621e704236ca2546914070